src/form-elements/FormElementCompliance.tsx

```diff
@@ -0,0 +1,224 @@
+    handleNotesChange,
+  )
+  const [isShowingFiles, toggleIsShowingFiles] = useToggleComplianceChildren(
+    handleFilesChange,
+  )
+
+  const [isDirty, setIsDirty] = useBooleanState(false)
+
+  const filteredOptions = useFormElementOptions({
+    element,
+    value: typedValue.value,
+    onChange: handleValueChange,
+    onFilter: onConditionallyShowOption,
+  })
+
+  return (
+    <div className="cypress-compliance-element">
+      <FormElementLabelContainer
+        className={`ob-compliance ${!isEven ? 'even' : 'odd'}`}
+        id={id}
+        element={element}
+      >
+        <div className="ob-compliance__container">
+          <FormElementOptions options={element.options}>
+            <div className="buttons ob-buttons ob-buttons-radio cypress-radio-button-group">
+              {filteredOptions.map((option) => {
+                const isSelected = typedValue.value === option.value
+                return (
+                  <div className="ob-button-radio-container" key={option.value}>
+                    <OptionButton
+                      element={element}
+                      option={option}
+                      isSelected={isSelected}
+                      onClick={() => {
+                        setIsDirty()
+                        handleValueChange(element, option.value)
+                      }}
+                      className={clsx(
+                        'button ob-button ob-button__input ob-radio__button cypress-radio-button-control',
+                        {
+                          'is-primary': isSelected,
+                          'is-light': !isSelected,
+                        },
+                      )}
+                    />
+                  </div>
+                )
+              })}
+            </div>
+          </FormElementOptions>
+          {(isDirty || displayValidationMessage) && !!validationMessage && (
+            <div role="alert" className="has-margin-top-8">
+              <div className="has-text-danger ob-error__text cypress-validation-message">
+                {validationMessage}
+              </div>
+            </div>
+          )}
+          <div className="buttons ob-buttons ob-buttons-compliance cypress-compliance-button-group">
+            <ComplianceButton
+              isActive={isShowingNotes}
+              icon="notes"
+              onClick={toggleIsShowingNotes}
+              disabled={!typedValue.value}
+            >
+              Notes
+            </ComplianceButton>
+            <ComplianceButton
+              isActive={isShowingFiles}
+              icon="perm_media"
+              onClick={toggleIsShowingFiles}
+              disabled={!typedValue.value}
+            >
+              Media
+            </ComplianceButton>
+          </div>
+          {isShowingNotes && (
+            <div className="ob-compliance-child-element">
+              <FormElementTextarea
+                id={`${element.id}-notes`}
```

Comment:
```suggestion
                id={`${id}-notes`}
```

This needs to be concatenated with the `id` property passed to this component. Otherwise the `id` will not be unique for entries in a repeatable set.